packages/react-meteor-data/createContainer.jsx

```diff
@@ -26,7 +27,7 @@ export default function createContainer(options = {}, Component) {
   }
 
   /* eslint-disable react/prefer-es6-class */
-  return React.createClass({
+  return createClass({
```

Comment:
Same comment as above - we might want to change `createClass` to `createReactClass`.